🚨Do not merge until Oct 18 launch -  dbt Semantic Layer docs

mirnawong1 opened the pull request:
creating integrate w data tools folder and quickstart for SL docs. content needs refining

Launch 8.30 PST am

## Description & motivation
added  integrate w data tools folder and quickstart for SL docs
closes #1979 #1989 #1988 #1978 #1985 

Docs project: https://www.notion.so/dbtlabs/The-dbt-Semantic-Layer-public-preview-352a6a409b434953a383f5273718668c

## To-do before merge

- [x] Change the base branch from next to current for Oct 18th launch 9am pst per PM
- [ ] Ensure PR #56 is merged


## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [x ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ x] The page has been added to `website/sidebars.js`
- [x] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

matthewshaver commented on website/docs/docs/integrate/avail-sl-integrations.md:
```diff
@@ -0,0 +1,31 @@
+
+There are two categories of partner integrations: 
+
+&mdash; **dbt Semantic Layer partners** - This refers to a group of dbt Labs partners who have full end-to-end integrations with the dbt Semantic Layer and leverage the dbt Server. 
```
Will the emdash put a bullet point in the text or does it have to be a - ?

matthewshaver commented on website/docs/docs/integrate/avail-sl-integrations.md:
```diff
@@ -0,0 +1,31 @@
+
+&mdash; **dbt metrics ready partners** - This refers to all partners integrating with dbt metrics functionality but do not leverage the dbt Server.
+
+For information on the different integration partners, their documention and more, review our [dbt Semantic Layer integrations](https://www.getdbt.com/product/semantic-layer-integrations) page.
```
```suggestion
For information on the different integration partners, their documentation, and more, review our [dbt Semantic Layer integrations](https://www.getdbt.com/product/semantic-layer-integrations) page.
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
```
It might have more impact to choose just one of the sentences to end with an exclamation point.  Whichever point you think is more important to stress.

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+    
+:::
+
+The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project), for consistent self-service within downstream data tools like BI and metadata management solutions. 
```
```suggestion
The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions.
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl_architecture.png" title="dbt Semantic Layer architecture" />
+
+**What makes the dbt Semantic Layer different?**
```
Should we use an ### header for this rather than bold type?

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+**What makes the dbt Semantic Layer different?**
+
+The dbt Semantic Layer removes code duplication and inconsistency when it comes to your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same data definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invok ...
```
```suggestion
The dbt Semantic Layer removes code duplication and inconsistency regarding your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same data definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications.
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+- Use Snowflake data platfrom 
+- Install the metrics package version 0.3.2 or higher in your dbt project
+- Set up the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview) in the integrated tool to import metric definitions
+- Recommended - Review the dbt metrics page and Getting started with the dbt Semantic Layer blog
```
It might be beneficial to link to these pages again so readers don't have to scroll and find.

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+- Have a multi-tenant [dbt Cloud](https://cloud.getdbt.com/) Teams or Enterprise account. 
```
```suggestion
- Have a multi-tenant [dbt Cloud](https://cloud.getdbt.com/) Teams or Enterprise account
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+### Considerations
+
+Here are some important considerations to know about during public preview:
```
```suggestion
Some important considerations to know about during the public preview:
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+&mdash; **What?** Public preview provides early access to new features, is supported and production ready, but not priced yet. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release. There may also still be additions or modifications to product behavior. 
+
+&mdash; **When?** Public preview will end once the dbt Semantic Layer is available for GA. After GA, the dbt Semantic Layer will only be available to dbt Cloud **Team** and **Enterprise** plans.
```
```suggestion
&mdash; **When?** Public preview will end once the dbt Semantic Layer is generally available. After GA, the dbt Semantic Layer will only be available to dbt Cloud **Team** and **Enterprise** plans.
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+## Product architecture 
+
+The dbt Semantic Layer product architecture includes four primary components, and you can review that and its plan availability in more detail below:
```
```suggestion
The dbt Semantic Layer product architecture includes four primary components:
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+| --- | --- | :---: | :---: | :---: | --- |
+| **dbt metrics** | Allows you to define metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source, Core |
+| **dbt Server**| HTTP server that is able to quickly compile metric queries per environment using dbt project code. | ✅ | ✅ | ✅ | BSL |
+| **SQL Proxy** | Reverse-proxy that accepts dbt-SQL (SQL + Jinja like query models and metrics, use macros), compiles the query into pure SQL, and executes the query against the data platform. | ✅ <br></br>_* Available during public preview only_ | ✅ | ✅ | Proprietary, Cloud (Team & Enterprise) |
```
"Available during public preview only". What happens when the public preview is over?

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+- Well-defined (the definition is agreed upon throughout the entire organization)
+- Time-bound (able to be compared across time)
+
+A great example of this is **revenue** - it can be aggregated on multiple levels (weekly, monthly, etc) and is key for the broader business to understand:
```
```suggestion
A great example of this is **revenue** &mdash it can be aggregated on multiple levels (weekly, monthly, etc) and is key for the broader business to understand:
```

matthewshaver commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+### Design and define metrics
+
+You can define your metrics in `.yml` files nested under a metrics key. To start designing or defining your own metrics in your dbt project, review the following documents to get started:
```
```suggestion
You can define your metrics in `.yml` files nested under a metrics key. To start designing or defining your own metrics in your dbt project, review the following documents:
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+:::
+
+Try out the features of the dbt Semantic Layer and follow this guide to learn more.
```
It might be a little redundant to put this before the intro.  Might read a little better if we include it in the intro or remove it.

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+## Design and define metrics in your dbt project
+
+Before you define metrics in your dbt project, you'll need to understand how to structure and organize your metrics. For more info on that and to understand the best practices, review our [Structuring and designing your metrics](url) blog post first.
```
```suggestion
Before you define metrics in your dbt project, you'll need to understand how to structure and organize your metrics. For more info on that and to understand the best practices, review our [Structuring and designing your metrics](URL) blog post first.
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+Now that's you've organized your metrics folder and files, you can define your metrics in `.yml` files nested under a `metrics` key.  
+
+1. Add the metric definitions found in the [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop_metrics) example to your dbt project. For example, to add an Expenses metric, see the below metrics you can define directly in your metrics folder: 
```
```suggestion
1. Add the metric definitions found in the [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop_metrics) example to your dbt project. For example, to add an Expenses metric, reference the following metrics you can define directly in your metrics folder: 
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+1. Click **Save** and **Compile** the code.
+2. Commit and merge the code changes that contain the metric definitions.
+3. If you'd like to further design and design your own metrics, review the following documentation:
```
```suggestion
3. If you'd like to further design(and design your own metrics), review the following documentation:
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+## Develop and query metrics
+
+You can dynamically develop and query metrics directly in dbt and verify their accuracy *before* running a job in the deployment environment by using the `metrics.calculate` macro and `metrics.develop` macro.
```
```suggestion
You can dynamically develop and query metrics directly in dbt and verify their accuracy *before* running a job in the deployment environment by using the `metrics.calculate` and `metrics.develop` macros.
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+    - [dbt metrics](/docs/building-a-dbt-project/metrics) will povide you in-depth detail on attributes, properties, filters, and how to define and query metrics.
+   
+    - [Structure and design your metric definition](URL) to understand best practices for designing and structuring metrics in your dbt project.
```
```suggestion
    - Visit the [Structure and design your metric definition](URL) page to understand best practices for designing and structuring metrics in your dbt project.
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+:::info 📌 
+
+Note, you will need access to dbt Cloud and the dbt Semantic Layer from your integrated tool of choice. 
```
```suggestion
**Note:** you will need access to dbt Cloud and the dbt Semantic Layer from your integrated tool of choice. 
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+
+:::info 📌
+
+Before you continue with the below steps, you must have a multi tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview).
```
```suggestion
Before continuing, you must have a multi-tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL but will not be able to browse dbt metrics in external tools, which requires access to the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview).
```

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+2. You won't be able to see your token again so we recommend you copy it somewhere safe.
+3. Go to **Deploy** and then **Environment**, and select your **Deployment** environment.
+4. In the upper right side of the page, click **Settings** and then **Edit**.
+5. Select dbt Version 1.2 (latest) or higher.
```
At the time we publish this will 1.2 still be the *latest* version?  Might be beneficial to leave "latest" out so we don't have to come back and edit this later.

matthewshaver commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,296 @@
+<details>
+  <summary>Ephemeral Models - Object does not exist or is not authorized</summary>
+  <div>
+    <div>Metrics cannot be defined on <a href="/docs/building-a-dbt-project/metrics">ephemeral models</a> and this is because we reference the underlying table in the query that generates the metric so we need the table/view to exist in the database. If your table/view does not exist in your database, you'll likely see this error message:
```
```suggestion
    <div>Metrics cannot be defined on <a href="/docs/building-a-dbt-project/metrics">ephemeral models</a> because we reference the underlying table in the query that generates the metric so we need the table/view to exist in the database. If your table/view does not exist in your database, you'll likely see this error message:
```

matthewshaver commented on website/docs/docs/integrate/setup-dbt-semantic-layer.md:
```diff
@@ -0,0 +1,72 @@
+
+:::info 📌
+
+Before you continue with the below steps, you **must** have a multi tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API.
```
```suggestion
Before you continue with the following steps, you **must** have a multi-tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API.
```

matthewshaver commented on website/docs/docs/integrate/setup-dbt-semantic-layer.md:
```diff
@@ -0,0 +1,72 @@
+
+:::
+
+You can set up the dbt Semantic Layer in dbt Cloud at the environment level by following the below steps:
```
```suggestion
You can set up the dbt Semantic Layer in dbt Cloud at the environment level by following these steps:
```

mirnawong1 commented:
thank you much @matthewshaver !

callum-mcdata commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+   * Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API.
+- Have both your production and development environments running dbt version 1.2 (latest) or higher
+- Use Snowflake data platfrom 
+- Install the [dbt metrics package](https://hub.getdbt.com/dbt-labs/metrics/latest/) version 0.3.2 or higher in your dbt project
```
Can we version this section and the dbt version? I think it could be confusing and lead to version mismatch if they try and use 0.3.2 with dbt core 1.3

Looks like we did so lower down but not up here.

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
```
Suggestion: Should say "in Public Preview."

We should make all instances of the term “Public Preview” be in title case, with capital P’s. This is a proper noun referring to a specific product state with specific definitions unique to dbt Labs’ product life cycle, not a common noun.

azzam34 replied:
Should say "reduce" code duplication and inconsistency, not remove.

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
+
+
+For more info on public preview and plan availability, check out the [public preview](/docs/integrate/quickstart-semantic-layer#public-preview) and [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
```
Should say "Public Preview."

Can we also link the upcoming dbt product states doc from here?
https://docs.google.com/document/d/1zy8R4NdBCp3TPWEQh0srXukqj45Zt7wZWOn1CAIEl6E/edit

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+:::
+
+
+## Public preview
```
Public Preview

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+## Public preview
+    
+We're excited to announce the dbt Semantic Layer is currently available for public preview, which means:
```
Public Preview

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+    
+We're excited to announce the dbt Semantic Layer is currently available for public preview, which means:
+
+&mdash; **Who?** The dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise) during public preview. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API. For more info on plan availability, check out our [product architecture](/docs/integrate/dbt-sema ...
```
Public Preview

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+&mdash; **Who?** The dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise) during public preview. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API. For more info on plan availability, check out our [product architecture](/docs/integrate/dbt-sema ...
+
+&mdash; **What?** Public preview provides early access to new features, is supported and production ready, but not priced yet. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release. There may also still be additions or modifications to product behavior. 
```
Public Preview

azzam34 replied:
Suggest matching the language at [this doc](https://docs.google.com/document/d/1zy8R4NdBCp3TPWEQh0srXukqj45Zt7wZWOn1CAIEl6E/edit):

> Public Previews are meant to provide early access to new features. The Semantic Layer is stable and can be considered for production deployments, but there may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality that is not backwards compatible. Technical support is provided, and relevant service level objectives (SLOs) apply. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release.

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+&mdash; **What?** Public preview provides early access to new features, is supported and production ready, but not priced yet. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release. There may also still be additions or modifications to product behavior. 
+
+&mdash; **When?** Public preview will end once the dbt Semantic Layer is available for GA. After GA, the dbt Semantic Layer will only be available to dbt Cloud **Team** and **Enterprise** plans.
```
Public Preview

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+&mdash; **When?** Public preview will end once the dbt Semantic Layer is available for GA. After GA, the dbt Semantic Layer will only be available to dbt Cloud **Team** and **Enterprise** plans.
+
+&mdash; **Where?** Public preview is enabled at the account level so you don’t need to worry about enabling it per user.
```
Public Preview

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+&mdash; **Where?** Public preview is enabled at the account level so you don’t need to worry about enabling it per user.
+
+&mdash; **Why?** Public preview is designed to test the functionality and collect feedback from the community on performance, usability, and documentation. 
```
Public Preview

azzam34 replied:
I think we can drop this - it's covered in my suggestion on Line 26.

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+- Have a multi-tenant [dbt Cloud](https://cloud.getdbt.com/) Teams or Enterprise account. 
```
Seems contradictory to say "Teams or Enterprise account" here, but then say Developer accounts can also use it in the next line. I would recommend changing this to just say multi-tenant dbt Cloud account.

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+Before you define metrics in your dbt project, you'll need to understand how to structure and organize your metrics. For more info on that and to understand the best practices, review our [Structuring and designing your metrics](URL) blog post first.
+
+Now that's you've organized your metrics folder and files, you can define your metrics in `.yml` files nested under a `metrics` key.  
```
*Now that

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+## Run your production job
+
+Once you’ve defined metrics in your dbt project, you can perform a job run in your deployment environment to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer at this moment. 
```
Might be better to say "Only the deployment environment is supported for the dbt Semantic Layer at this moment."

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+Once you’ve defined metrics in your dbt project, you can perform a job run in your deployment environment to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer at this moment. 
+
+1. Go to **Deploy** in the navigation and select **Jobs** to re-run the job with the most recent code in the deployment environment.
+2. Your metric should appear as red nodes in the dbt Cloud IDE and dbt directed acyclic graphs (DAG).
```
*a red node

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+## Set up dbt Semantic Layer
+
+Before continuing, you must have a multi-tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL but will not be able to browse dbt metrics in external tools, which requires access to the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview).
```
Same comment as above. This should just say "multi-tenant dbt Cloud plan", because the following sentence then says that Developer plans can also use the Semantic Layer, just with a couple of functionality limitations.

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+
+Before continuing, you must have a multi-tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL but will not be able to browse dbt metrics in external tools, which requires access to the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview).
+
+To query your universally-defined metrics in your integration tool, you need to [set up the dbt Semantic Layer](/docs/integrate/setup-dbt-semantic-layer#set-up-dbt-semantic-layer) in dbt Cloud to connect with your integration tool:
```
Recommend rephrasing to "To query your dbt metrics in an integrated partner tool"

azzam34 replied:
"integration tool" sounds off to me

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+<details>
+  <summary>How are you storing my data?</summary>
+  <div>
+    <div>dbt does not store your data, we temporarily ingest the data from the data warehouse, pass it to connecting tool, and then drop the data.</div>
```
recommend "cloud data platform" instead of data warehouse

azzam34 commented on website/docs/docs/integrate/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,292 @@
+  <div>
+    <div>Some components of the dbt Semantic Layer are open source like dbt-core, the dbt_metrics package, and the BSL licensed dbt-server. The dbt Proxy Server (what is actually compiling the dbt code) and the Metadata API are not open source. <br></br>
+      
+During public preview, the dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise). <br></br>
```
*Public Preview

azzam34 commented on website/docs/docs/integrate/setup-dbt-semantic-layer.md:
```diff
@@ -0,0 +1,72 @@
+sidebar_label: "Set up the dbt Semantic Layer"
+---
+
+With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more. Configure the dbt Semantic Layer in dbt Cloud to connect with your integration tool. 
```
See my comment on the phrasing in this section in the Quickstart above

azzam34 commented on website/docs/docs/integrate/setup-dbt-semantic-layer.md:
```diff
@@ -0,0 +1,72 @@
+
+Before you set up the dbt Semantic Layer, make sure you meet the following:
+
+- Have a multi-tenant [dbt Cloud](https://cloud.getdbt.com/) Teams or Enterprise account. 
```
See my comment on the phrasing in this section in the Quickstart above

azzam34 commented on website/docs/docs/integrate/setup-dbt-semantic-layer.md:
```diff
@@ -0,0 +1,72 @@
+
+## Set up dbt Semantic Layer
+
+Before you continue with the following steps, you **must** have a multi-tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API.
```
See my comment on the phrasing in this section in the Quickstart above

azzam34 commented on website/docs/docs/integrate/avail-sl-integrations.md:
```diff
@@ -0,0 +1,28 @@
+
+There are two categories of partner integrations: 
+
+-  **dbt Semantic Layer partners** - This refers to a group of dbt Labs partners who have full end-to-end integrations with the dbt Semantic Layer and leverage the dbt Server. 
```
Recommend: "Semantic Layer integrations are capable of interactively querying dbt metrics, surfacing the underlying data in partner tools."

That's the copy we're using on this landing page too:
https://www.getdbt.com/product/semantic-layer-integrations

azzam34 commented on website/docs/docs/integrate/avail-sl-integrations.md:
```diff
@@ -0,0 +1,28 @@
+
+-  **dbt Semantic Layer partners** - This refers to a group of dbt Labs partners who have full end-to-end integrations with the dbt Semantic Layer and leverage the dbt Server. 
+
+-  **dbt metrics ready partners** - This refers to all partners integrating with dbt metrics functionality but do not leverage the dbt Server.
```
Should be "Metrics Ready partners"

azzam34 replied:
Recommend: "Metrics Ready integrations facilitate building, discovery, and collaboration on dbt metric definitions."

That matches the copy we're using on this landing page, and gives it a slightly more positive framing:
https://www.getdbt.com/product/semantic-layer-integrations

azzam34 commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,116 @@
+
+The dbt Semantic Layer is currently available for public preview. With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
+
+For more info on public preview and plan availability, check out the [public preview](/docs/integrate/dbt-semantic-layer#public-preview) and [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
```
*Public Preview

azzam34 commented on website/docs/docs/integrate/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,116 @@
+
+## Public preview
+
+The dbt Semantic Layer is currently available for public preview, which means:
```
See my comments on this in the Quickstart section

JKarlavige commented on website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,133 @@
+
+<VersionBlock firstVersion="1.3" >
+
+<span>&#8226;</span> Have a North American multi-tenant <a href="https://cloud.getdbt.com/">dbt Cloud</a> account. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API. <br />
+<span>&#8226;</span> Have both your production and development environments running dbt version 1.3 or higher <br />
+<span>&#8226;</span> Use Snowflake data platform <br />
+<span>&#8226;</span> Install the <a href="https://hub.getdbt.com/dbt-labs/metrics/latest/">dbt metrics package</a> version <code>">=1.3.0", "<1.4.0"</code> in your dbt project <br />
+<span>&#8226;</span> Set up the <a href="https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview">Metadata API</a> in the integrated tool to import metric definitions <br />
+<span>&#8226;</span> Recommended - Review the <a href="https://docs.getdbt.com/docs/building-a-dbt-project/metrics">dbt metrics page</a> and <a href="https://docs.getdbt.com/blog/getting-started-with-the-dbt-semantic-layer">Getting started with the dbt Semantic Layer</a> blog <br />
```
One of the build failures is coming from the `<span>&#8226;</span>` blocks. If intending to use a list here, all `<span>&#8226;</span>` should be replaced with `-`. The `<br />` tags are not needed as each list item will drop to a new line on its own. I would also adjust the links here to use the `[link_name](link_url)` markdown format rather than html, as well as use the ` ` symbols for code blocks rather than `<code>`.

For example, the first bullet in this block would be updated to look like:
```
 - Have a North American multi-tenant [dbt Cloud](https://cloud.getdbt.com/) account. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API.
```

JKarlavige replied:
All `<span>&#8226;</span>` within this PR has to be updated to use the `-` symbol

JKarlavige commented on website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,133 @@
+
+</VersionBlock>
+    
+<---!
+<Snippet src="sl-prerequisites" />
+--->
```
Disregard my original comment, it still fails with the updated comment. If the Snippet shouldn't be used here, it should be removed from the page. Commenting it out with `<!-- <Snippet src="sl-prerequisites" /> -->` doesn't look to work either

JKarlavige commented on website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md:
```diff
@@ -0,0 +1,91 @@
+
+</VersionBlock>
+
+<---!
+<Snippet src="sl-prerequisites" />
+--->
```
Same for this comment

JKarlavige commented on website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md:
```diff
@@ -0,0 +1,308 @@
+
+<b>Fix:</b>
+
+<span>&#8226;</span> You will need to materialize the model that the metric is built on as a table/view/incremental.</div>
```
The `</div>` at the end of this line is also causing an issue, it is closing the list item with `</div>` instead of `</li>` thus missing a closing tag. This would be updated to:

```suggestion
- You will need to materialize the model that the metric is built on as a table/view/incremental.

</div>
```

JKarlavige replied:
Note how the `</div>` is bumped down two lines

JKarlavige commented on website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md:
```diff
@@ -44,29 +44,25 @@ To use the dbt Semantic Layer, you’ll need to meet the following:
-    
-<---!
-<Snippet src="sl-prerequisites" />
---->
```
Just note i removed these snippets as the `<---!` is causing an error. If the snippets should be used here they will have to be readded

bennieregenold7 commented on website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md:
```diff
@@ -0,0 +1,132 @@
+
+&mdash; **Who?** The dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise) during Public Preview. Team and Enterprise accounts will be able to set up the Semantic Layer and [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview) in the integrated partner tool to import metric definition. Developer accounts will be able to query the Proxy Server using ...
+
+&mdash; **What?** Public Previews provide early access to new features. The Semantic Layer is stable and you can use it for production deployments, but there may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality that is not backwards compatible. dbt Labs provides support, and relevant service level ...
```
There's an extra word in the last line.

> We will introduce pricing for the dbt Semantic Layer will alongside the General Available (GA) release.

I think the word `will` before the word `alongside` should be removed.

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/develop-in-the-cloud.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-features.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-features.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-features.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-tips.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/develop/dbt-cloud-features.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

mirnawong1 committed:
Update version-control-basics.md

mirnawong1 committed:
Update version-control-basics.md

mirnawong1 committed:
Update git-version-control.md

mirnawong1 committed:
Update dbt-cloud-features.md

mirnawong1 committed:
Update dbt-cloud-features.md

mirnawong1 committed:
Update dbt-cloud-tips.md

mirnawong1 committed:
Update dbt-cloud-tips.md

mirnawong1 committed:
Update _redirects

mirnawong1 committed:
Update sidebars.js

mirnawong1 committed:
Update and rename website/docs/docs/develop/dbt-cloud-tips.md to website/docs/docs/get-started/dbt-cloud-tips.md

mirnawong1 committed:
Update and rename website/docs/docs/develop/develop-in-the-cloud.md to website/docs/docs/get-started/develop-in-the-cloud.md

mirnawong1 committed:
Update _redirects

mirnawong1 committed:
Update and rename website/docs/docs/develop/dbt-cloud-features.md to website/docs/docs/get-started/dbt-cloud-features.md

mirnawong1 committed:
Rename website/docs/docs/develop/source-install.md to website/docs/docs/get-started/source-install.md

mirnawong1 committed:
Rename website/docs/docs/develop/pip-install.md to website/docs/docs/get-started/pip-install.md

mirnawong1 committed:
Rename website/docs/docs/develop/installation-overview.md to website/docs/docs/get-started/installation-overview.md

mirnawong1 committed:
Rename website/docs/docs/develop/homebrew-install.md to website/docs/docs/get-started/homebrew-install.md

mirnawong1 committed:
Rename website/docs/docs/develop/docker-install.md to website/docs/docs/get-started/docker-install.md

mirnawong1 committed:
Rename website/docs/docs/develop/connection-profiles.md to website/docs/docs/get-started/connection-profiles.md

mirnawong1 committed:
Rename website/docs/docs/develop/about-the-cli.md to website/docs/docs/get-started/about-the-cli.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/building-your-first-project/build-your-first-models.md to website/docs/docs/get-started/getting-started/build-your-first-models.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/building-your-first-project/schedule-a-job.md to website/docs/docs/get-started/getting-started/building-your-first-project/schedule-a-job.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/building-your-first-project/test-and-document-your-project.md to website/docs/docs/get-started/getting-started/building-your-first-project/test-and-document-your-project.md

mirnawong1 committed:
Rename website/docs/docs/get-started/getting-started/build-your-first-models.md to website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/getting-set-up/setting-up-bigquery.md to website/docs/docs/get-started/getting-started/getting-set-up/setting-up-bigquery.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/getting-set-up/setting-up-databricks.md to website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/getting-set-up/setting-up-redshift.md to website/docs/docs/get-started/getting-started/getting-set-up/setting-up-redshift.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/getting-set-up/setting-up-snowflake.md to website/docs/docs/get-started/getting-started/getting-set-up/setting-up-snowflake.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/learning-more/refactoring-legacy-sql.md to website/docs/docs/get-started/learning-more/refactoring-legacy-sql.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/learning-more/using-jinja.md to website/docs/docs/get-started/learning-more/using-jinja.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/add-a-seed.md to website/docs/docs/get-started/getting-started/add-a-seed.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/building-your-first-project.md to website/docs/docs/get-started/getting-started/building-your-first-project.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/create-a-project.md to website/docs/docs/get-started/getting-started/create-a-project.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/getting-set-up.md to website/docs/docs/get-started/getting-started/getting-set-up.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/getting-started-dbt-cloud.md to website/docs/docs/get-started/getting-started-dbt-cloud.md

mirnawong1 committed:
Rename website/docs/docs/develop/getting-started/getting-started-dbt-core.md to website/docs/docs/get-started/getting-started-dbt-core.md

mirnawong1 committed:
Update _redirects

Commit:
fixing merge conflicts

Commit:
tweak copy to maintain equal lengh of cards

mirnawong1 committed:
Update website/docs/docs/dbt-versions/product-lifecycles.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 committed:
Merge pull request #2174 from dbt-labs/mwong-lifecyle-region-new-pages

Add product lifecycle page and regions page

Commit:
tweak copy to maintain equal lengh of cards

Commit:
tweak copy to maintain equal lengh of cards

Commit:
fixing conflicts

runleonarun committed:
Merge branch 'next' into mwong-IDE-IA-launch

runleonarun committed:
Update website/docs/docs/get-started/getting-started/getting-set-up.md

jasnonaz committed:
Merge branch 'next' into tighten-join-copy

jasnonaz committed:
Merge pull request #2210 from dbt-labs/tighten-join-copy

tweak copy to maintain equal lengh of cards

mirnawong1 committed:
Merge branch 'next' into mwong-IDE-IA-launch

JKarlavige committed:
manually trigger preference center

mirnawong1 committed:
Update dbt-cloud-features.md

mirnawong1 committed:
Update develop-in-the-cloud.md

mirnawong1 committed:
Update dbt-cloud-features.md

JKarlavige committed:
Merge pull request #2213 from dbt-labs/ot-cookie-settings-fix

manually trigger preference center

mirnawong1 committed:
Update dbt-cloud-features.md

mirnawong1 committed:
Update dbt-cloud-features.md

mirnawong1 committed:
Update dbt-cloud-features.md

mirnawong1 committed:
Update dbt-support.md

adding dbt core support blurb

mirnawong1 committed:
Merge branch 'next' into mwong-IDE-IA-launch

runleonarun committed:
Merge branch 'next' into mwong-dbt-cli-support

breezyfasano committed:
Change link on slack icon in footer

mirnawong1 committed:
Merge pull request #2187 from dbt-labs/mwong-IDE-IA-launch

IDE content changes for IA launch

runleonarun committed:
Update website/docs/docs/dbt-support.md

breezyfasano committed:
Merge branch 'next' into footer-slack

breezyfasano committed:
Merge pull request #2215 from dbt-labs/footer-slack

Change link on slack icon in footer

mirnawong1 committed:
Update website/docs/docs/dbt-support.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'next' into mwong-dbt-cli-support

Commit:
Merge branch 'current' into next

mirnawong1 committed:
Merge branch 'next' into mwong-dbt-cli-support

mirnawong1 committed:
Merge pull request #2214 from dbt-labs/mwong-dbt-cli-support

Update dbt-support.md

nghi-ly committed:
Merge branch 'current' into next

breezyfasano committed:
Fix 2 cardsZ

nghi-ly committed:
Clean up duplicate content and stale pages

nghi-ly committed:
Merge branch 'next' into ly-docs-clean-up-pages

nghi-ly committed:
Missed sidebar entry

nghi-ly committed:
Merge branch 'ly-docs-clean-up-pages' of github.com:dbt-labs/docs.getdbt.com into ly-docs-clean-up-pages

breezyfasano committed:
Merge pull request #2216 from dbt-labs/card-fix

Fix 2 cards

nghi-ly committed:
Merge branch 'next' into ly-docs-clean-up-pages

joellabes committed:
Create help feed embeds for a handful of pages (#2195)

* Create help feed embeds for a handful of pages

* Remove sneaky backtick

* Missed a backtick

* Maybe fix tags not being respected?

* Differentiate between loading failure and no posts. HelpFeed shows all time

* Assume no messages, not error

* limit returned discourse topics to 3

* Remove forum links when there are already FAQs

* Merge branch 'next' into discourse-help-feed-embeds

Co-authored-by: john-rock <johnmrock.jr@gmail.com>

nghi-ly committed:
Update sidebar label

breezyfasano committed:
Remove rogue console log

nghi-ly committed:
Update website/docs/docs/get-started/getting-started/getting-set-up.md

breezyfasano committed:
Merge pull request #2221 from dbt-labs/console-fix

Remove rogue console log

nghi-ly committed:
Merge pull request #2220 from dbt-labs/ly-docs-minor-sidebar-fix

Update sidebar label

runleonarun committed:
Merge branch 'next' into ly-docs-clean-up-pages

nghi-ly committed:
Move two pages

nghi-ly committed:
Merge branch 'current' into next

nghi-ly committed:
Update _redirects

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

nghi-ly committed:
Overview page on deployments

nghi-ly committed:
Fold in review comment

runleonarun committed:
Merge branch 'next' into ly-docs-move-pages

nghi-ly committed:
Merge branch 'next' into ly-docs-clean-up-pages

runleonarun committed:
Update admin-cloud-api.md (#2224)

nghi-ly committed:
Merge pull request #2222 from dbt-labs/ly-docs-move-pages

Move two pages

matthewshaver committed:
Update python-models.md

nghi-ly committed:
Resolve merge conflict

nghi-ly committed:
Merge branch 'ly-docs-clean-up-pages' of github.com:dbt-labs/docs.getdbt.com into ly-docs-clean-up-pages

matthewshaver committed:
Update website/docs/docs/build/models.md

matthewshaver committed:
Update website/docs/docs/build/python-models.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

matthewshaver committed:
Update python-models.md

nghi-ly committed:
Merge branch 'next' into ly-docs-deployment-overview

nghi-ly committed:
Merge pull request #2217 from dbt-labs/ly-docs-clean-up-pages

Clean up duplicate content and stale pages

runleonarun committed:
Merge branch 'next' into ia-python

runleonarun committed:
Updating the What is dbt? page (#2207)

* updating Intro

* new linge

* update

* adding power features

* Apply suggestions from code review

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

Co-authored-by: runleonarun <leona@dbtlabs.com>
Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

runleonarun committed:
Merge branch 'next' into ia-python

runleonarun committed:
Update website/docs/docs/build/python-models.md

joellabes committed:
Fix community links

matthewshaver committed:
Merge pull request #2206 from dbt-labs/ia-python

Matt's Python edits and moves for IA restructure

nghi-ly committed:
Add redirect and updated links

joellabes committed:
Update dbt cloud apis links

joellabes committed:
Merge branch 'next' into joel-docs-broken-links-community

nghi-ly committed:
Merge branch 'next' into ly-docs-deployment-overview

joellabes committed:
trigger ci

joellabes committed:
Merge branch 'joel-docs-broken-links-community' of https://github.com/dbt-labs/docs.getdbt.com into joel-docs-broken-links-community

joellabes committed:
Put back missing period

matthewshaver committed:
Correcting broken links

nghi-ly committed:
Merge pull request #2226 from dbt-labs/joel-docs-broken-links-community

Joel docs broken links community and api docs

nghi-ly committed:
Update website/docs/docs/build/custom-schemas.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

nghi-ly committed:
Fold in review comments

nghi-ly committed:
Merge branch 'ly-docs-deployment-overview' of github.com:dbt-labs/docs.getdbt.com into ly-docs-deployment-overview

nghi-ly committed:
Update website/docs/docs/deploy/deployment-overview.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

nghi-ly committed:
Update website/docs/docs/deploy/deployment-overview.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

nghi-ly committed:
Merge branch 'next' into ly-docs-deployment-overview

nghi-ly committed:
Update website/docs/docs/deploy/deployment-overview.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

nghi-ly committed:
Merge pull request #2225 from dbt-labs/ly-docs-deployment-overview

Overview page on deployments

matthewshaver committed:
Link fixes

matthewshaver committed:
Link fixes

matthewshaver committed:
More link fixes

Commit:
fixing merge conflicts

matthewshaver committed:
Update website/docs/docs/dbt-cloud/cloud-ide/ide-beta.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

runleonarun committed:
fixing links (#2228)

Co-authored-by: runleonarun <leona@dbtlabs.com>

matthewshaver committed:
Update website/docs/docs/dbt-cloud/cloud-ide/ide-beta.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/dbt-cloud/cloud-ide/ide-beta.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

matthewshaver committed:
Matts moves (#2219)

* Merging cloud upgrade pages

* Combines these two pages: 

Generating documentation in dbt Cloud - 
(https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-generating-documentation)

Viewing docs in the IDE - 
https://docs.getdbt.com/docs/dbt-cloud/cloud-ide/viewing-docs-in-the-ide

* Redirects for viewing docs in IDE

* More additions to the IA changes
Redirects, retitle of manage-access, and sidebar update

* Added CLI information

* Update and rename manage-access.md to about-access.md

* Update website/sidebars.js

* Update _redirects

* Update website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

* Update website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

* Update website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

* Update website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

* Update website/docs/docs/collaborate/manage-access/about-access.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

* Update cloud-build-and-view-your-docs.md

* Update website/docs/docs/collaborate/manage-access/about-access.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

nghi-ly committed:
Fix links

matthewshaver committed:
Merge branch 'next' into matts-links-4

matthewshaver committed:
Merge pull request #2231 from dbt-labs/matts-links-4

Link fixes for Next branch

runleonarun committed:
Merge branch 'next' into ly-docs-links

nghi-ly committed:
Merge pull request #2232 from dbt-labs/ly-docs-links

Fix links

nghi-ly committed:
Fix links 2

nghi-ly committed:
Merge branch 'next' into ly-docs-fix-links-2

nghi-ly committed:
Merge pull request #2233 from dbt-labs/ly-docs-fix-links-2

Fix links 2

runleonarun committed:
fixing links (#2234)

* fixing links

* Update website/docs/docs/get-started/getting-started/getting-set-up/setting-up-snowflake.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

Co-authored-by: runleonarun <leona@dbtlabs.com>
Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

nghi-ly committed:
Fix links from npm output

nghi-ly committed:
Merge pull request #2236 from dbt-labs/ly-docs-moar-fix-links

Fix links from npm output

runleonarun committed:
mo' links (#2237)

Co-authored-by: runleonarun <leona@dbtlabs.com>

joellabes committed:
Add leading slash for /img path

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Merge branch 'next' into mwong-SL-launch

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update dbt-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update dbt-semantic-layer.md

mirnawong1 committed:
Update set-dbt-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update dbt-semantic-layer.md

Commit:
fixing conflicts and praying

Commit:
fixing conflicts and praying

runleonarun commented on website/docs/docs/collaborate/cloud-build-and-view-your-docs.md:
```diff
@@ -31,7 +30,7 @@ You configure project documentation to generate documentation when the job you s
 5. Click **Save**.
 
-## Generating documentation
+## Generating documentation 
```
```suggestion
## Generating documentation
```

runleonarun committed:
Update website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

runleonarun commented on website/docs/docs/dbt-versions/upgrade-core-in-cloud.md:
```diff
@@ -11,7 +11,7 @@ In dbt Cloud, both jobs and environments are configured to use a specific versio
 Navigate to the settings page of an environment, then click **edit**. Click the **dbt Version** dropdown bar and make your selection. From this list, you can select an available version of Core to associate with this environment.
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/Environment-settings.png" title="Settings of a dbt Cloud environment"/>
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/Environment-settings.png" title="Settings of a dbt Cloud environment"/> 
```
```suggestion
<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/Environment-settings.png" title="Settings of a dbt Cloud environment"/>
```

runleonarun committed:
Update website/docs/docs/dbt-versions/upgrade-core-in-cloud.md

runleonarun commented on website/docs/docs/get-started/dbt-cloud-tips.md:
```diff
@@ -7,7 +7,7 @@ sidebar_label: "dbt Cloud tips"
 # dbt Cloud tips
 
-The Cloud IDE is fully equipped with keyboard shortcuts, features, and development tips to help you to work like a dbt Cloud power user!  Use this Cloud IDE cheat sheet to help you save time and quickly reference. 
+The Cloud IDE is fully equipped with keyboard shortcuts, features, and development tips to help you to work like a dbt Cloud power user!  Use this Cloud IDE cheat sheet to help you save time and quickly reference.
```
```suggestion
The Cloud IDE is fully equipped with keyboard shortcuts, features, and development tips to help you to work like a dbt Cloud power user!  Use this Cloud IDE cheat sheet to help you save time and quickly reference. 
```

runleonarun commented on website/docs/docs/get-started/dbt-cloud-tips.md:
```diff
@@ -31,7 +31,7 @@ Double-click a node in the directed acyclic graph (DAG) to open that file in a n
 ## IDE Keyboard shortcuts
 
-There are default keyboard shortcuts that can help make development more productive and easier for everyone. 
+There are default keyboard shortcuts that can help make development more productive and easier for everyone.
```
```suggestion
There are default keyboard shortcuts that can help make development more productive and easier for everyone. 
```

runleonarun commented on website/docs/docs/get-started/dbt-cloud-tips.md:
```diff
@@ -72,4 +72,3 @@ There are default keyboard shortcuts that can help make development more product
 ## Related docs
 
 - [Guides](/docs/get-started/getting-started/overview)
```
```suggestion
- [Getting started](/docs/get-started/getting-started/overview)
- [Develop in the Cloud](/docs/get-started/develop-in-the-cloud)
```

runleonarun committed:
Apply suggestions from code review

runleonarun commented on website/docs/docs/introduction.md:
```diff
@@ -65,3 +65,4 @@ As a dbt user, your main focus will be on writing models (i.e. select queries) t
 - [Best practice guides](/guides/best-practices)
 - [What is a dbt Project?](/docs/build/projects)
 - [dbt run](/docs/get-started/run-your-dbt-projects)
+
```
```suggestion
```

runleonarun committed:
Update website/docs/docs/introduction.md

runleonarun committed:
Merge branch 'current' into mwong-SL-launch

runleonarun commented on website/docs/docs/deploy/dashboard-status-tiles.md:
```diff
@@ -5,6 +5,7 @@ description: "Embed Status Tiles in your dashboards to provide consumers with co
 ---
 
 ## Overview
+
```
```suggestion
```

runleonarun committed:
Update website/docs/docs/deploy/dashboard-status-tiles.md

runleonarun committed:
Merge branch 'current' into mwong-SL-launch

mirnawong1 committed:
Merge branch 'current' into mwong-SL-launch

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update avail-sl-integrations.md

mirnawong1 committed:
Update connect-github.md

mirnawong1 committed:
Merge branch 'current' into mwong-SL-launch

mirnawong1 committed:
Update dbt-global-variables.js

mirnawong1 committed:
Merge branch 'current' into mwong-SL-launch

mirnawong1 committed:
Update dbt-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update set-dbt-semantic-layer.md

mirnawong1 committed:
Update avail-sl-integrations.md

mirnawong1 committed:
Merge branch 'current' into mwong-SL-launch

nghi-ly committed:
Merge branch 'current' into mwong-SL-launch

mirnawong1 committed:
Update dbt-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update dbt-semantic-layer.md

mirnawong1 committed:
Update set-dbt-semantic-layer.md

mirnawong1 committed:
Update avail-sl-integrations.md

mirnawong1 committed:
Update set-dbt-semantic-layer.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update avail-sl-integrations.md

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update dbt-semantic-layer.md

mirnawong1 committed:
Merge branch 'current' into mwong-SL-launch

mirnawong1 committed:
Update quickstart-semantic-layer.md

mirnawong1 committed:
Update set-dbt-semantic-layer.md
